lib/v2/tencent/news/author.js

```diff
@@ -5,35 +5,33 @@ const timezone = require('@/utils/timezone');
 
 module.exports = async (ctx) => {
     const mid = ctx.params.mid;
-    const response = await got(`https://pacaio.match.qq.com/om/mediaArticles?mid=${mid}&num=10&page=0`);
-    const reponse = response.data;
-    const title = reponse.mediainfo.name;
-    const description = reponse.mediainfo.intro;
-    const list = reponse.data;
+    const homePageInfoUrl = `https://i.news.qq.com/i/getUserHomepageInfo?chlid=${mid}`;
+    const userInfo = await ctx.cache.tryGet(homePageInfoUrl, async () => (await got(homePageInfoUrl)).data.userinfo);
+    const title = userInfo.nick;
+    const description = userInfo.user_desc;
+    const suid = encodeURIComponent(userInfo.suid);
 
-    const items = await Promise.all(
-        list.map((item) => {
-            const title = item.title;
-            const pubDate = timezone(parseDate(item.time), +8);
-            const itemUrl = item.vurl;
-            const author = item.source;
-            const abstract = item.abstract;
+    const items = (await got(`https://i.news.qq.com/getSubNewsMixedList?guestSuid=${suid}&tabId=om_index`)).data.newslist.map((item) => {
```

Comment:
Please set the 3rd parameter of `tryGet` to `config.cache.routeRxpire` and the 4th parameter to `false` otherwise `newlist` won't receive any updates.
docs: https://docs.rsshub.app/joinus/use-cache.html